src/avatar.cpp

```diff
@@ -1461,6 +1461,19 @@ bool avatar::invoke_item( item *used, const std::string &method )
     return Character::invoke_item( used, method );
 }
 
+void avatar::update_cardio_acc()
+{
+    const int prev_cardio_acc = get_cardio_acc();
+    const int last_24h_kcal = calorie_diary.begin()->spent;
+    int adjustment = 0;
+    if( prev_cardio_acc > last_24h_kcal ) {
+        adjustment = std::sqrt( prev_cardio_acc - last_24h_kcal );
+    } else if( last_24h_kcal > prev_cardio_acc ) {
+        adjustment = -std::sqrt( last_24h_kcal - prev_cardio_acc );
+    }
+    set_cardio_acc( prev_cardio_acc + adjustment );
```

Comment:
```suggestion
    if( prev_cardio_acc > last_24h_kcal ) {
        adjustment = -std::sqrt( prev_cardio_acc - last_24h_kcal );
    } else if( last_24h_kcal > prev_cardio_acc ) {
        adjustment = std::sqrt( last_24h_kcal - prev_cardio_acc );
    }
    set_cardio_acc( prev_cardio_acc + adjustment );
```
If the previous is above the amount spent, the cardio_acc should go down, not up, and vice-versa.

Reply:
good catch